apps/public-docsite-v9/src/Concepts/Upgrade/FromV8/Components/RadioGroup.stories.mdx

```diff
@@ -0,0 +1,197 @@
+```jsx
+import * as React from 'react';
+import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react/lib/ChoiceGroup';
+import { mergeStyles } from '@fluentui/react/lib/Styling';
+import { CatIcon } from '@fluentui/react-icons-mdl2';
+
+const ChoiceGroupCustomOptionExample = () => {
+  const optionRootClass = mergeStyles({ display: 'flex', alignItems: 'center', gap: '5px' });
+
+  const options: IChoiceGroupOption[] = [
+    {
+      key: 'A',
+      text: 'A label with an icon',
+      ariaLabel: 'A label with a cat icon',
+      onRenderField: (props, render) => {z
+        return (
+          <div className={optionRootClass}>
+            {render!(props)}
+            <CatIcon />
+          </div>
+        );
+      },
+    },
+    { key: 'B', text: 'Option B', styles: { root: { border: '1px solid green' } } },
+    { key: 'C', text: 'Option C', disabled: true },
+    { key: 'D', text: 'Option D' },
+  ];
+
+  return <ChoiceGroup defaultSelectedKey="B" options={options} label="Pick one" />;
+}
+```
+
+An equivalent `RadioGroup` implementation:
+
+```jsx
+import * as React from 'react';
+import { Label, Radio, RadioGroup } from '@fluentui/react-components/unstable';
+import { useId } from '@fluentui/react-utilities';
+import { makeStyles, shorthands } from '@griffel/react';
+import { AnimalCat24Regular } from '@fluentui/react-icons';
+
+const useIconOptionStyles = makeStyles({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    ...shorthands.gap('5px'),
+  },
+});
+
+const useLabelStyles = makeStyles({
+  root: {
+    display: 'flex',
+    ...shorthands.gap('5px'),
+  },
+});
+
+const useGreenBorderOptionStyles = makeStyles({
+  root: {
+    ...shorthands.border('1px', 'solid', 'green'),
+  },
+});
+
+const RadioGroupCustomOptionExample = () => {
+  const labelId = useId('label-');
+  const iconOptionStyles = useIconOptionStyles();
+  const labelStyles = useLabelStyles();
+  const greenBorderOptionStyles = useGreenBorderOptionStyles();
+
+  return (
+    <>
+      <Label id={labelId} required>
+        Pick One
+      </Label>
+      <RadioGroup aria-labelledby={labelId} defaultValue="B">
+        <div className={iconOptionStyles.root}>
+          <Radio
+            value="A"
+            label={{
+              className: labelStyles.root,
+              children: (
+                <>
+                  A <AnimalCat24Regular />
+                </>
+              ),
+            }}
+          />
+        </div>
+        <Radio value="B" label="B" className={greenBorderOptionStyles.root} />
+        <Radio value="C" label="C" disabled />
+        <Radio value="D" label="D" />
+      </RadioGroup>
+    </>
+  );
+};
+```
+
+## Props Tables
+
+This table maps v8 `ChoiceGroup` props to the v9 `RadioGroup` equivalent.
+
+| Purpose                                                             | v8                   | v9                                 | Details                                                                      |
```

Comment:
Given that the caller is upgrading from v8 to v9 probably has existing v8 code, it might be overkill to document the purpose of every property.  It will add a lot more effort to the other upgrade guides and we would have to keep this in sync with the existing v8 and v9 documentation for the properties.